patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -7441,6 +7441,27 @@
  		int pickedProjectileId = -1;
  		if (PickAmmo_TryFindingSpecificMatches(sItem.type, item.type, out pickedProjectileId))
  			projToShoot = pickedProjectileId;
+@@ -41015,18 +_,8 @@
+ 			projToShoot = 117;
+ 
+ 		if (projToShoot == 42) {
+-			if (item.type == 370) {
+-				projToShoot = 65;
+-				Damage += 5;
+-			}
+-			else if (item.type == 408) {
+-				projToShoot = 68;
+-				Damage += 5;
+-			}
+-			else if (item.type == 1246) {
+-				projToShoot = 354;
+-				Damage += 5;
+-			}
++			projToShoot = ItemID.Sets.SandgunAmmoToProjectile[item.type];
++			Damage += ItemID.Sets.SandgunAmmoDamage[item.type];
```

Comment:
It's preferred to use multi-line comments (`/* ... */`) to remove large swathes of code. This minimizes exposure of ReLogic code in patches and makes them more resilient to change.

Reply:
Ah, alright. Couldn't find anything about this specifically in the guidelines page (might've missed it though). I'll change it to comment out the code instead of just deleting it.